numpy/polynomial/chebyshev.py

```diff
@@ -1830,6 +1831,77 @@ def chebcompanion(c):
     return mat
 
 
+def chebinterp(func, deg, x=None, args=(), window=None):
```

Comment:
Not aimed at @uoaei here, but...

Why do we call the functions these things? Why is `np.polynomial.chebyshev.chepinterp` a useful alias for `np.polynomial.Chebyshev,interp`? Can we start only adding functions in the latter form?